Auth0/Authentication.swift

```diff
@@ -1114,43 +1126,49 @@ public extension Authentication {
                challenge: PasskeyLoginChallenge,
                connection: String? = nil,
                audience: String? = nil,
+               organization: String? = nil,
```

Comment:
The parameter here is in a different order than in the protocol declaration. Therefore, the function in this extension is not defining a default implementation of the declared function, but an entirely new function instead.

Please make sure the function signature in the extension matches the one in the protocol declaration.